lib/active_merchant/billing/gateways/paypal.rb

```diff
@@ -73,6 +73,11 @@ def build_sale_or_authorization_request(action, money, credit_card_or_referenced
         xml.target!
       end
 
+      def api_version(options)
+        return API_VERSION_3DS2 if options.dig(:three_d_secure, :version) =~ /^2/
```

Comment:
could we pass them the real version? (aka just `options.dig(:three_d_secure, :version)`?)

Reply:
or
```ruby
options.dig(:three_d_secure, :version) || API_VERSION
```

Reply:
never mind, it's API versions, not 3DS version 👓